tsec-libsodium/src/main/scala/tsec/libsodium/kx/kx.scala

```diff
@@ -0,0 +1,42 @@
+package tsec.libsodium
+
+import cats.evidence.Is
+import tsec.common.HKByteArrayNewt
+
+package object kx {
+
+  private[tsec] val SodiumKey$$ : HKByteArrayNewt = new HKByteArrayNewt {
```

Comment:
We already have a `SodiumKey`.

We should maybe reuse classes from asymmetric crypto.